contentprovider.go

```diff
@@ -637,7 +666,7 @@ func (p *contentProvider) candidateMatchScore(ms []*candidateMatch, language str
 			} else if startMatch || endMatch {
 				addScore("EdgeSymbol", (scoreSymbol+scorePartialSymbol)/2)
 			} else {
-				addScore("InnerSymbol", scorePartialSymbol)
+				addScore("OverlapSymbol", scorePartialSymbol)
```

Comment:
I like how this change is so simple. Switching from Inner -> Overlap makes sense to me conceptually.

Reply:
Some dark arts, but should we consider innersymbol worth more than overlapsymbol? Hard to imagine a query where you get both innersymbol and overlapsymbol so likely not worth it.